enrolments/models.py

```diff
@@ -1,3 +1,14 @@
 from django.db import models
 
-# Create your models here.
+
+class Enrolment(models.Model):
+    family = models.ForeignKey("registration.Family", on_delete=models.PROTECT)
+    # class = models.ForeignKey()
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    created_by = models.ForeignKey(
+        "accounts.User",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+    )
```

Comment:
Do we wanna have maybe a field to say if the enrolment is active or not? Because families can un-enroll right?

Reply:
smort 🧠 yes let's add a boolean named `active` – @annoyingpr feel free to do that in a later PR if you'd like